internal/injector/injector_test.go

```diff
@@ -35,6 +35,8 @@ type testConfig struct {
 	SyntheticReferences map[string]typed.ReferenceKind `yaml:"syntheticReferences"`
 	GoLang              context.GoLangVersion          `yaml:"required-lang"`
 	Code                string                         `yaml:"code"`
+	ImportPath          string                         `yaml:"import-path"`
+	ImportMap           map[string]string              `yaml:"import-map"`
```

Comment:
This can probably just be a `[]string`... This is just a set, and modeling as a `map[string]string` is heavier than necessary

```suggestion
	ImportMap           []string              `yaml:"import-map"`
```

Reply:
You are right this is a set so a `map[string]struct{}` could be enough. But since a `map[string]string` is already filled for other purposes earlier in the code, doing one less copy of the map seemed more compelling.

Reply:
This is the YAML for specifying test expectations... I feel like "comfort of specifying tests" prevails here...